yarn.lock

```diff
@@ -2744,6 +2744,10 @@
   version "0.0.0"
   uid ""
 
+"@kbn/securitysolution-t-grid@link:bazel-bin/packages/kbn-securitysolution-t-grid":
+  version "0.0.0"
```

Comment:
is version `0.0.0` expected here?

Reply:
it matches with the other packages who have just been created